api/plugins/bingo-elastic/python/bingo_elastic/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = "1.4.1"
```

Comment:
Do we need to set this manually, as well as in Poetry?

Reply:
It will be cool, could you add it to the CI pipeline?